### PR TITLE
fix(deps): update ember to 3.16.5

### PR DESCRIPTION
**Pull request by fkm**
https://github.com/emberjs/ember.js/pull/18831

### CODE REVIEW

**Comment by anehx** [comment]
Did you do this with `ember-cli-update`?

**Comment by czosel** [comment]
I wanted to ask exactly the same question :sweat_smile:

**Comment by fkm** [comment]
I changed it manually. Running the cli now.

**Comment by fkm** [comment]
Done

File: tests/dummy/app/resolver.js [hunk 1]
```diff
@@ -1,3 +0,0 @@
-import Resolver from "ember-resolver";
-
-export default Resolver;
```
Comment (czosel) [inline tests/dummy/app/resolver.js]: do you know why we had this in the first place? (cc @anehx)

Comment (anehx) [reply tests/dummy/app/resolver.js]: Default ember blueprint

Comment (czosel) [reply tests/dummy/app/resolver.js]: Ah, thanks :upside_down_face:

**Pull request merged**